src/ontology/mondo-ingest.Makefile

```diff
@@ -598,6 +598,19 @@ update-jinja-sparql-queries:
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_replace_annotation_based_mappings.py
 	python3 $(SCRIPTSDIR)/ordo_mapping_annotations/create_sparql__ordo_mapping_annotations_violation.py
 
+
+#################
+##### Ad hoc ####
+#################
+tmp/ordo-subsets.tsv: $(COMPONENTSDIR)/ordo.owl
```

Comment:
## Updated goal to depend on component
Addresses:
- https://github.com/monarch-initiative/mondo-ingest/issues/511#issuecomment-2094305764